automation/taskcluster/upload_apk_nimbledroid.py

```diff
@@ -17,7 +17,7 @@
 
 def uploadApk(apk,key):
 	headers = {"Accept":"*/*"}
-	payload = {'auto_scenarios':'false'}
+	payload = {'auto_scenarios':'false', 'device_config': 'android7'}
```

Comment:
@emalysz One question: will this run on both Android 7 and Android 5 or just Android 7? afaik, we'd want to do both Android 7 & 5.

Reply:
Added additional flag so that both are now used